src/FunctionApps/python/tests/test_fhir.py

```diff
@@ -0,0 +1,114 @@
+import unittest
+
+from unittest import mock
+from IntakePipeline import fhir
+
+import json
+import os
+
+
+class TestFhir(unittest.TestCase):
```

Comment:
the unittest classes are kind of the older way to do things here - we've been on `pytest` for all of the other function apps - were you able to get that running?

tl;dr though is that these tests should be individual functions in `tests/IntakePipeline/fhir_test.py` with the other tests for that module

Reply:
I was able to get pytest running in my environment, and confirmed that it was picking up these tests written as they are.  I'll change them though to make them consistent with other tests.

Reply:
Test cases have been updated.